include/onnx-mlir/Compiler/OMCompilerMacros.h

```diff
@@ -0,0 +1,29 @@
+/*
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+//====----------------- Compiler.h - Compiler abstraction support ---------===//
+//
+// Copyright 2022 The IBM Research Authors.
+//
+// =============================================================================
+//
+// This file defines several macros, which allows use of compiler-specific
+// features in a way that remains portable. This header can be included from
+// either C or C++.
+//
+//===----------------------------------------------------------------------===//
+
+#ifndef ONNX_MLIR_COMPILER_MACROS_H
+#define ONNX_MLIR_COMPILER_MACROS_H
+
+/// OM_EXTERNAL_VISIBILITY - classes, functions, and variables marked with this
```

Comment:
ONNX_MLIR_EXPORT is already defined in OnnxMlirCompier.h. I don't think we need two definitions, so please either use ONNX_MLIR_EXPORT everywhere, or change the definition of OM_EXTERNAL_VISIBILITY, so that it can be used instead of ONNX_MLIR_EXPORT

Reply:
I tried to use ONNX_MLIR_EXPORT but got this failure: https://dev.azure.com/onnx-pipelines/onnx/_build/results?buildId=22626&view=logs&j=4060f429-1919-5440-1fce-77a91697330b&t=4cbc8f84-8349-586a-8281-73be4740c2c3, and had no Windows environment to debug.

The definition of ONNX_MLIR_EXPORT is more complicated than I expected and I am not totally understand. Really appreciate if someone from your team could help with this issue by a follow-up PR.

Reply:
@tungld @sstamenova I will create a PR to clean up the include stuff and hopefully make the ONNX_MLIR_EXPORT work for both Compiler and Runtime libraries.

Reply:
Thank you @gongsu832!